phpBB/docs/events.md

```diff
@@ -46,6 +46,18 @@ acp_email_find_username_append
 * Since: 3.1.7-RC1
 * Purpose: Add content at the end of the fimd username link
 
+acp_ext_list_disabled_name_after
```

Comment:
Modified the location of these during the merge. Please make sure to order these alphabetically next time.